front/src/common/Pathfinding/Pathfinding.tsx

```diff
@@ -281,8 +285,12 @@ function Pathfinding({ mustUpdate = true, zoomToFeature }: PathfindingProps) {
   const [isInfraLoaded, setIsInfraLoaded] = useState(false);
   const [reloadCount, setReloadCount] = useState(1);
   const [isInfraError, setIsInfraError] = useState(false);
-  const [postPathfinding] = osrdEditoastApi.usePostPathfindingMutation();
-  const { data: infra } = osrdEditoastApi.useGetInfraByIdQuery(
+
+  const [isInitialized, setIsInitialized] = useState(false);
```

Comment:
Maybe to clarify because there is also the infra in the component
```suggestion
  const [isPathfindingInitialized, setIsPathfindingInitialized] = useState(false);
```